docs/install-upgrade/supported-devices.md

```diff
@@ -1,26 +1,6 @@
 # Supported Devices
 
-You can find detailed information about devices in the [Switch Profiles Catalog](../reference/profiles.md) and in the
+You can find detailed information about devices in the [Switch Catalog](../reference/profiles.md) and in the
 User Guide [switch features and port naming](../user-guide/profiles.md).
 
-## Spine
-
-* Celestica DS3000
-* Celestica DS4000
-* Dell S5232F-ON
-* Dell Z9332F-ON
-* Edgecore DCS204 (AS7726-32X)
-* Edgecore DCS501 (AS7712-32X-EC)
-* Supermicro SSE-C4632SB
-
-## Leaf
-
-(could be used for collapsed-core)
-
-* Celestica DS3000
-* Dell S5232F-ON
-* Dell S5248F-ON
-* Edgecore DCS203 (AS7326-56X)
-* Edgecore DCS204 (AS7726-32X)
-* Edgecore EPS203 (AS4630-54NPE)
-* Supermicro SSE-C4632SB
+--8<-- "docs/install-upgrade/supported-devices.md.gen"
```

Comment:
This line was probably inserted by mistake?

Reply:
No, it's embedding the `supported-devices.md.gen` that is generated in the fabric repo

Reply:
OK my bad, sorry! :slightly_smiling_face: